### PR TITLE
1st approach restaurant

### PR DESCRIPTION
**Pull request by gapinal**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+    font-size: 14px;
+    margin-right: 30px;
+    font-weight: 600;
+    color: #fff;
+    letter-spacing: 1px;
+    text-decoration: none;
+    background-color: transparent;
+    box-sizing: border-box;
+}
+.segment-one button {
```
Comment (nmicht) [inline css/styles.css]: Is better to assign class names related to the content instead the position or the styling

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+    background-color: #e51a94;
```
Comment (nmicht) [inline css/styles.css]: Use the colors as variables

File: index.html [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Events</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="css/styles.css"/>
+    <link rel="stylesheet" href="node_modules/font-awesome/css/font-awesome.min.css">
+    <link href="images/favicon.png" rel="icon">
+    <script src="main.js"></script>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous">
```
Comment (nmicht) [inline index.html]: You should use font-awesome as a dependency, from node_modules

File: index.html [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+            <span>RESTAURANTE + JUEGOS DE MESA</span>    
+        <div> <button type="button" name="button">HAZ UNA RESERVACIÓN</button></div>
+        </div>
+    </div>
+    <div class="discount">
+        <div class="content">
+            <span>¡Obtén tu cupón de DESCUENTO!</span>
+            <input type="Nombre" placeholder="Nombre*" name="name" id="">
+            <input type="email" placeholder="Email*" name="email" id="">
+            <button type="button" name="button">ENVIAR</button>
```
Comment (nmicht) [inline index.html]: Remove the `name` attribute from the button

File: index.html [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+        </div>
+    </div>
+    <div class="segment-two">
+        <div class="content">
+            <h2>NOSOTROS</h2> 
+            <p>El mejor Restaurante dedicado al espacio lúdico de Juegos de Mesa. Contamos con los mejores juegos de mesa, tienda, expertos en diversión y un menú deliciosamente divertido. 
+            <br><br>El mejor Restaurante dedicado al espacio lúdico de Juegos de Mesa. Contamos con los mejores juegos de mesa, tienda. expertos en diversión y un menú deliciosamente divertido.
```
Comment (nmicht) [inline index.html]: Do not use `br`. In this case is better to have two paragraphs and margin between them.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+    </div>
+    <div class="segment-two">
+        <div class="content">
+            <h2>NOSOTROS</h2> 
+            <p>El mejor Restaurante dedicado al espacio lúdico de Juegos de Mesa. Contamos con los mejores juegos de mesa, tienda, expertos en diversión y un menú deliciosamente divertido. 
+            <br><br>El mejor Restaurante dedicado al espacio lúdico de Juegos de Mesa. Contamos con los mejores juegos de mesa, tienda. expertos en diversión y un menú deliciosamente divertido. 
+            </p>    
+            <div><button type="button" name="button">RESERVA</button></div>            
+        </div>
+        <div><img src="images/people.png" alt=""></div>
```
Comment (nmicht) [inline index.html]: Always always have the `alt` attribute with information related to the image

File: index.html [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+            <input type="Nombre" placeholder="Celular*" name="cel" id="">
+            <input type="Nombre" placeholder="Fecha*" name="name" id="">
+            <input type="Nombre" placeholder="Hora*" name="name" id="">
+            <input type="Nombre" placeholder="Comentarios*" name="name" id="">
+            <button type="button" name="button">QUIERO MI RESERVACIÓN</button>                     
+            </ul> 
+        </div>
+    </div>
+    <div class="footer-one">
+        <div class="content">
```
Comment (nmicht) [inline index.html]: For the contact information will be great to use `address` tag
See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address

File: index.html [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+            <div>
+                <img class="" src="images/icon_phone.png">
+                <span><br>TELÉFONO<br><br></span>
+                <span>3629·4947</span>
```
Comment (nmicht) [inline index.html]: You can use the telephone with a link to make a call directly
See https://css-tricks.com/the-current-state-of-telephone-links/

File: package.json [hunk 9]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "gapinal_portfolio",
+  "version": "1.0.0",
+  "description": "this is my own personal portfolio",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/gapinal/proyecto_final.git"
+  },
+  "author": "Georgia Andrea Pinal",
```
Comment (nmicht) [inline package.json:9]: Include your email also
```
"author": {
    "name": "asdfasdf",
    "email": "asdfasdfasd"
}
```

File: package.json [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/gapinal/proyecto_final/issues"
```
Comment (nmicht) [inline package.json:12]: Update the url with your github url

**Review by nmicht — CHANGES_REQUESTED** [review]
I left a lot of comment on your code.
Please review, fix and update

**Comment by gapinal** [comment]
media query

**Comment by gapinal** [comment]
media query done last pull request

**Pull request closed without merging**